docs/book/form-creation/using-annotations.md

```diff
@@ -6,84 +6,101 @@ representation for it, and potentially a hydrator for mapping the form elements
-  `target_object` is the element to compose, `is_collection` flags this as a
-  collection, and `options` can take an array of options to pass into the
-  collection.
-- `ErrorMessage`: specify the error message to return for an element in the case
-  of a failed validation. Expects a string value.
-- `Exclude`: mark a property to exclude from the form or fieldset. This
-  annotation does not require a value.
-- `Filter`: provide a specification for a filter to use on a given element.
-  Expects an associative array of values, with a "name" key pointing to a string
-  filter name, and an "options" key pointing to an associative array of filter
-  options for the constructor: `@Filter({"name": "Boolean", "options": {"casting":true}})`.
-  This annotation may be specified multiple times.
-- `Flags`: flags to pass to the fieldset or form composing an element or
-  fieldset; these are usually used to specify the name or priority. The
-  annotation expects an associative array: `@Flags({"priority": 100})`.
-- `Hydrator`: specify the hydrator class to use for this given form or fieldset.
-  A string value is expected.
-- `InputFilter`: specify the input filter class to use for this given form or
-  fieldset. A string value is expected.
-- `Input`: specify the input class to use for this given element. A string value
-  is expected.
-- `Instance`: specify an object class instance to bind to the form or fieldset.
-- `Name`: specify the name of the current element, fieldset, or form. A string
-  value is expected.
-- `Options`: options to pass to the fieldset or form that are used to inform
-  behavior &mdash; things that are not attributes; e.g. labels, CAPTCHA adapters,
-  etc. The annotation expects an associative array: `@Options({"label": "Username:"})`.
-- `Required`: indicate whether an element is required. A boolean value is
-  expected. By default, all elements are required, so this annotation is mainly
-  present to allow disabling a requirement.
-- `Type`: indicate the class to use for the current element, fieldset, or form.
-  A string value is expected.
-- `Validator`: provide a specification for a validator to use on a given
-  element. Expects an associative array of values, with a "name" key pointing to
-  a string validator name, and an "options" key pointing to an associative array
-  of validator options for the constructor:
-  `@Validator({"name": "StringLength", "options": {"min":3, "max": 25}})`.
-  This annotation may be specified multiple times.
-- `ContinueIfEmpty`: indicate whether the element can be submitted when it
-  is empty. A boolean value is expected. If `@Required` is set to `false`, this
-  needs to be set to `true` to allow the field to be empty.
-
-To use annotations, include them in your class and/or property docblocks.
+Annotations allow us to solve this problem. With v3, laminas-form supports two
+different types of annotations: PHP8 attributes and traditional PHPDoc annotations.
```

Comment:
> PHPDoc

We use "DocBlock" or "docblock" in all documentations, the preferred type is "DocBlock" as is used in [phpDocumentor documentation](https://phpdoc.org).